hack/install.sh

```diff
@@ -529,6 +529,19 @@ do_install() {
 	exit 1
 }
 
+while getopts "e:" opt; do
+	case $opt in
+		e)
+			environment=$OPTARG
+			;;
+	esac
+done
+
+if [ ! -z "$environment" ] && [ $environment = "AzureChinaCloud" ]; then
```

Comment:
`$environment` should be quoted here, and it might actually make sense to convert this to a `case` for simplicity, ie:
```bash
case "$environment" in
    AzureChinaCloud)
        ....
        ;;
esac
```